src/utils/notification.js

```diff
@@ -1,33 +1,43 @@
 import Notiflix from 'notiflix';
 
-const notifyUserRegisterSuccess = () => {
-  Notiflix.Report.success(
-    'Registration Success!',
-    'Now You have an account in Wallet.',
-    'Have fun!',
-  );
+const notifyUserEmailSenTSuccess = email => {
+  Notiflix.Report.success('Verification email sent!', `Check :${email} Inbox.`, 'OK!');
 };
 
-const notifyRegistrationFailure = () => {
+const notifyUserEmailSenTFailure = message => {
   Notiflix.Report.failure(
-    'Registration Fail!ed.',
-    'This User already exist or too short Username / e-mail / password (each must have at least 7 characters).',
+    'Sending the verification email failed.',
+    `Message from server: ${message}.`,
     'Try again',
   );
 };
 
-const notifyLoginFailure = () => {
-  Notiflix.Report.failure(
-    'Something goes wrong.',
-    'Do You have an account? Check if Username and Password are correct. Please try again or contact our service.',
-    'Try again',
+const notifyUserEmailVerifiedSuccess = firstName => {
+  Notiflix.Report.success(
+    `Hi, ${firstName}! Email verified!`,
```

Comment:
Email verified -> Your email address has been successfully verified!

Reply:
Nie, musi być krótka wiadomość, bo nie zawija tekstu ten notiflix.

Reply:
To chociaż 'Email is verified' ?